packages/shared/src/components/GrowthBookProvider.tsx

```diff
@@ -80,7 +80,8 @@ export const GrowthBookProvider = ({
           'AES-CBC',
           128,
         ).then((features) => {
-          gb.setFeatures(JSON.parse(features));
+          console.log(features);
+          // gb.setFeatures(JSON.parse(features));
```

Comment:
This was returning an invalid JSON string for me locally, will need some support debugging this as I don't have direct access to Grouth Book

Reply:
We can maybe look into the SDK to see if there is a local overwrite available